styles/_description-lists.scss

```diff
@@ -17,14 +17,25 @@ dl {
   }
 
   &.inline {
-    @extend .govuk-grid-row;
+    dt {
+      float: left;
+      margin-right: 5%;
+      max-width: 30%;
+    }
+
+    dd {
+      margin-left: 35%;
+      margin-bottom: $govuk-gutter;
```

Comment:
The bottom margin should be applied to the `dt` as well so that there is always margin irrespective of which is longer.

As this is, a `dt` which wraps where it's `dd` does not will result in no margin being visible.